README.md

```diff
@@ -41,22 +47,496 @@ Now, the client has to include the Mikros SDK into their gaming product and setu
 6. *(Optional)* Mikros collects and sends metadata for better user insights and is also crucial for Analytics purpose. This can be enabled or disabled by the ***Auto Track User Metadata*** option from the Mikros Settings. By default, it is kept enabled and is recommended to keep it that way.
 
 <a name="sample-app-&-documentation"></a>
-## Sample App & Dcumentation
+## Sample App & Documentation
 
-* Authentification
+<a name="authentication"></a>
+* Authentication
     
     
- This is the Mikros Sample App main screen. The Authentication button will allow you to be authenticated and allow you to use the Mikros Sample App.
- 
- ![MIKROS MAIN SCREEN, add from Git URL dropdown](Documentation/Mikros-main-screen.png)
- 
- After clicking the Authentication button you will be presented with 3 options. (Sign-in, Sign-up, and Sign-out)
- 
- 
- After clicking the Sign-up button you will  be presented with this screen and will need a Username, Email, and Password to sign up.
- 
- 
- After clicking the Sign-out button you will be presented with this message. Letting you know you have signed out of the Mikros Sample App.
+This is the Mikros Sample App main screen. The Authentication button will allow you to be authenticated and allow you to use the Mikros Sample App.
+
+![MIKROS MAIN SCREEN, add from Git URL dropdown](Documentation/Mikros-main-screen.png)
```

Comment:
@Velite12 always user lowercasing for images. Rename the image Mikros-main-screen.png -> `mikros_main_screen.png` and update references. Do the same for all images e.g. Mikros-sign-in.png -> `mikros_signin.png` | Mikros-sign-up.png -> `mikros_signup.png` | Mikros-sign-out.png -> `mikros_signout.png` 

Note: In code we treat signin, signup, signout as one word. In business language we use two. It is a matter of convenience.